### DISCUSSION TITLE
How to add parent prims for exported materials when using Arnold Export?

### DISCUSSION THREAD

**Discussion by frankchen211** [Autodesk/arnold-usd#1651] [Q&A]
If it is possible to have parent prims to hold Material primss just like MayaUsd Export - all materials are under `mtl` prim by default.

**Comment by sebastienblor** [comment]
Sorry, I'm not sure to understand. Do you want materials to be under "mtl" or you don't want that ? 
Since a few months we are always exporting materials under a "mtl" scope, like MayaUSD does, to be consistent with the industry.

Which version are you testing ?
